test/dynamo/test_functions.py

```diff
@@ -1252,6 +1252,14 @@ def test_partials_lambda(x):
         triple = functools.partial(multiply, y=3)
         return triple(x)
 
+    def test_add_dtype(self):
```

Comment:
@eellison Is there already a dynamo test for add where a configuration could be added vs a new test?

Reply:
maybe near `test_add_const_int` in `test_torchinductor`

Reply:
Thanks @janeyx99, @eellison. I've changed the test to use a dtype in `test_add_const_int`.